db/migrate/20190826105224_create_help_texts.rb

```diff
@@ -0,0 +1,24 @@
+class CreateHelpTexts < ActiveRecord::Migration
+  def up
+    create_table :help_texts do |t|
+      t.string :key, null: false
```

Comment:
Ich würde hier auch noch das model ablegen, damit im besser zwischen den HelpTexts unterschieden werden kann.

Reply:
Vielen Dank für das Review @amaierhofer :-) Finde ich eine gute Idee, den `key` in `model`, `action` und `attribute` zu splitten. Ich habe mich hier initial mal für einen I18n-mässigen `key` entschieden, um mir noch alle Optionen offen zu halten: Wie im Issue #811 beschrieben bin ich mir nicht sicher, ob die bisher implementierten "Orte" (Seiten und Formular-Felder) alles abdecken, wofür die Hilfetexte gebraucht werden können. Vielleicht gibt es irgendwann auch beispielsweise Bedarf für mehrere Hilfetexte pro Seite ("extension"-mässig) oder globale Hilfetexte für die Navigationen etc.

Denkst Du, wir sollten nochmals konzeptionell anschauen, was hier (in Zukunft) gefordert ist (ich kann kurz gerne bei euch vorbeischauen) oder ist das nicht so wichtig im Moment und ich soll einfach mal eine der beiden Varianten umsetzen?